src/server/rpc/procedures/dev/dev.js

```diff
@@ -3,6 +3,6 @@ const rpcUtils = require('../utils/index');
 module.exports = {
     isSupported: () => process.env.ENV !== 'production',
     echo: function (argument){
-        return rpcUtils.jsonToSnapList(argument);
+        return argument;
```

Comment:
Not sure why this changed... It looks like this is part of a dependent PR....

Reply:
merged in from a dependent PR. After that one is merged and there is a merge conflict I'll take care of this one.